release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -328,6 +328,41 @@ In the table below, features are marked with the following statuses:
 [id="ocp-4-6-known-issues"]
 == Known issues
 
+* In {product-title} 4.1, anonymous users were able to access discovery endpoints. Later releases revoked this access to reduce the possible attack surface for security exploits because some discovery endpoints are forwarded to aggregated API servers. However, unauthenticated access is preserved in upgraded clusters so that existing use cases are not broken.
++
+Cluster administrators for {product-title} 4.6 clusters that have been upgraded from {product-title} 4.1 can either continue to allow unauthenticated access or can revoke unauthenticated access. It is recommended to revoke unauthenticated access unless there is a specific need for it. If you do continue to allow unauthenticated access, be aware of the increased risks.
```

Comment:
This sentence is a bit awkward. If it doesn't change the meaning, you might try rewording to something like:

> If you are a cluster administrator with clusters that have been upgraded from {product-title} 4.1 to 4.6, you can either revoke or continue to allow unauthenticated access.